batch/batch/server/globals.py

```diff
@@ -1,18 +1,42 @@
+import os
 import datetime
 import collections
 
+import hailjwt as hj
+
+from .google_storage import upload_private_gs_file_from_string, download_gs_file_as_string
+from .google_storage import exists_gs_file
+
+
+if 'GOOGLE_APPLICATION_CREDENTIALS' not in os.environ:
+    os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = '/batch-gsa-key/privateKeyData'
```

Comment:
This feels fishy to me as well. I think we talked about explicitly passing the credentials file to the google library?

Reply:
Done.